### PR TITLE
fix: remove extra curly braces from macro calls

### PR DESCRIPTION
**Pull request by caugner**
### Description

Removes extra curly braces before/after macro calls.

### Motivation

Macro calls require exactly two curly braces both before and after, but as part of https://github.com/mdn/yari/pull/10224 I noticed that there were two macro calls with more leading curly braces, which should result in a macro render error. Looking closer, I also found several occurrences with more trailing curly braces, one of which looks like it should have been a closing parenthesis.

### Additional details

See https://developer.mozilla.org/ko/docs/Web/API/MouseEvent/clientX for failed macro renders:

image

See https://developer.mozilla.org/es/docs/Web/HTTP/CSP#describiendo_una_pol%C3%ADtica for how the additional trailing curly braces render as content:

image

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
Approved for ES team.

Pinged you just in case you wanna check it by your self @Graywolf9

**Review by yin1999 — APPROVED** [review]
LGTM for zh-CN. Thank you @caugner

File: files/fr/learn/html/introduction_to_html/html_text_fundamentals/index.md
Comment (cw118) [inline files/fr/learn/html/introduction_to_html/html_text_fundamentals/index.md:834]: I took the opportunity to replace the macros by links since we prefer links for the french locale.

```suggestion
En HTML, nous utilisons l'élément [`<strong>`](/fr/docs/Web/HTML/Element/strong) (forte importance) comme balise de telles circonstances. En plus de rendre le document plus lisible, ces balises sont reconnues par les lecteurs d'écran et énoncées avec des intonations différentes. Par défaut, les navigateurs mettent le texte marqué en gras, mais il ne faut pas utiliser cette balise pour mettre en gras. Pour cela, utilisez un élément [`<span>`](/fr/docs/Web/HTML/Element/span) et du CSS, ou plus simplement un élément [`<b>`](/fr/docs/Web/HTML/Element/b) (voir ci-dessous).
```

**Review by cw118 — CHANGES_REQUESTED** [review]
There is just one small typo for fr. That being said, thank you @caugner !

File: files/ru/web/api/pushmanager/subscribe/index.md
Comment (leon-win) [inline files/ru/web/api/pushmanager/subscribe/index.md:25]: Approved for `ru`. Thank you! :+1:

**Review by leon-win — APPROVED** [review]

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja` locale. Thank you!

**Review by josielrocha — APPROVED** [review]
Approve for `pt-BR`

**Review by cw118 — APPROVED** [review]
Looks good, thank you so much! And happy new year everyone 🎉

**Comment by SphinxKnight** [comment]
If I understand correctly, we're now waiting for @mdn/yari-content-ko for the last approval.

**Review by hochan222 — APPROVED** [review]
Forgive me for being late. LGTM for `ko-locale`

Thank you so much :) Happy new year everyone 🎉

**Comment by SphinxKnight** [comment]
THanks a lot @hochan222 :)

**Pull request merged**
